mne/decoding/receptive_field.py

```diff
@@ -332,41 +310,12 @@ def _check_dimensions(self, X, y, predict=False):
         return X, y, X_dim, y_dim
 
 
-def _pad_time_series(X, n_delays, fill_mean=True):
-    """Return a zero- or mean-padded input time series.
-
-    Parameters
-    ----------
-    X : array, shape (n_times[, n_epochs], n_features)
-        The time series to pad.
-    n_delays : int
-        The number of delays.
-    fill_mean : bool
-        If True, the fill value will be the mean along the time dimension
-        of the feature. If False, the fill value will be zero.
-
-    Returns
-    -------
-    padded : array, shape(n_padded[, n_epochs], n_features)
-        The padded data, where ``n_padded = n_times + n_delays - 1``.
-    """
-    fill_value = 0
-    if fill_mean:
-        fill_value = np.mean(X, axis=0, keepdims=True)
-        if X.ndim == 3:
-            fill_value = np.mean(fill_value, axis=1, keepdims=True)
-    X = np.pad(X, ((0, n_delays - 1),) + ((0, 0),) * (X.ndim - 1), 'constant')
-    X[-(n_delays - 1):] = fill_value
-    return X
-
-
-def _delay_time_series(X, tmin, tmax, sfreq, newaxis=0, axis=0,
-                       epoch_axis=1, fill_mean=False):
+def _delay_time_series(X, tmin, tmax, sfreq, fill_mean=False):
     """Return a time-lagged input time series.
 
     Parameters
     ----------
-    X : array, shape (n_times[, n_epochs], n_features)
+    X : array, shape (n_times[, n_epochs][, n_features])
```

Comment:
I was thinking to require at the least `(n_times, n_features)` to avoid confusion about what the extra dimension means. E.g., if we allow for just `(n_times, n_epochs)` AND `(n_times, n_features)`, how do we know which the user wants if they give a 2D input?

Reply:
The code does already actually, I'll update the docstring